server/msteams/client.go

```diff
@@ -280,6 +280,23 @@ func (tc *ClientImpl) RefreshToken(token *oauth2.Token) (*oauth2.Token, error) {
 	return conf.TokenSource(context.Background(), token).Token()
 }
 
+func (tc *ClientImpl) GetAppCredentials(applicationID string) ([]clientmodels.Credential, error) {
+	application, err := tc.client.ApplicationsWithAppId(&applicationID).Get(tc.ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+	credentials := []clientmodels.Credential{}
+	credentialsList := application.GetPasswordCredentials()
+	for _, credential := range credentialsList {
+		credentials = append(credentials, clientmodels.Credential{
+			ID:         credential.GetKeyId().String(),
+			Name:       *credential.GetDisplayName(),
+			ExpireDate: *credential.GetEndDateTime(),
```

Comment:
nit: instead of `ExpireDate`, should we translate the field (and related APIs in the metrics) more directly as `EndDateTime`? I always like grepping that "finds everything" vs. having a translation mid-way that obscures things.

Reply:
Done